.husky/commit-msg

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
 
-yarn commitlint --edit $1
+npx commitlint --edit $1
```

Comment:
Changes on the `.husky` configuration folder are linked to the [given commit](https://github.com/ovh/manager/commit/b0677029ee8423d5ced97fd76d90d0438b41e1ab) that indicates `update yarn lock`.

**question:** Could you please indicate why such changes should comes with this particular commit?

Also, I'm having trouble to understand why we should use `npx` over `yarn`.

Reply:
It is very simple, we had an incompatibility between node v18.17.1 and husky v8.0.1. It's a way to resolve this problem

Reply:
Would it be possible to have more detail about the error?
I've tested locally both node@18.17.1 and husky@8.0.1 and didn't face any incompatibility issue.

Reply:
Its about a 'segmentation fault' error, all our team have this, when we try to commit

we alse find this common bug 
https://github.com/typicode/husky/issues/1246